bindings/binding_typescript_wasm/__tests__/__snapshots__/transform.js.snap

```diff
@@ -127,7 +141,12 @@ exports[`transform in strip-only mode should throw an error when it encounters a
   "endLine": 1,
   "filename": "test.ts",
   "message": "\`module\` keyword is not supported. Use \`namespace\` instead.",
-  "snippet": "module foo",
+  "snippet": "   
+        
+ 1   module foo { }
```

Comment:
Is it possibile to remove the 1?

Reply:
There’s no option to remove line numbers